docs/handbook/network-communications.md

```diff
@@ -331,6 +331,99 @@ You can create additional CIFS datasets using the following 4 commands.
 
 ### Hipster as an NFS server
 
+< Placeholder for introduction content >
+
+
+#### Manual pages
+
+You can read more about the commands and files used or mentioned with the `man` pages, and at the provided links in the document.
+
+* [dfstab(5)](https://illumos.org/man/5/dfstab)
+* [share_nfs(8)](https://www.illumos.org/man/8/share_nfs)
+* [share(8)](https://www.illumos.org/man/8/share)
+* [shareall(8)](https://www.illumos.org/man/8/shareall)
+* [svcadm(8)](https://www.illumos.org/man/8/svcadm)
+* [unshare(8)](https://www.illumos.org/man/8/share)
+
+For example to view the full documentation for `share(8)` you can use this command:
+```console
+$ man share.8
+```
+
+
+#### Sharing a folder with NFS
+
+To create a read-only temporary NFS share on an existing folder in this example `/nfs/example` you can use the following [share(8)](https://www.illumos.org/man/8/share) command:
+
+```console
+# share -d example-share -F nfs -o ro /nfs/example
+```
+
+The flags provided can be described in a table such as the following:
+
+|Flag|Description|
+|------|-----------|
+|-d|Adds a description for the share in this case `example-share`|
+|-F|Filesystem type, in this case NFS.|
+|-o|Options for the NFS share in this case `rw`|
```

Comment:
Looks like this should be `ro` based on the command above. Presumably meaning read-only.